project.clj

```diff
@@ -43,7 +45,9 @@
                  [org.slf4j/log4j-over-slf4j ~slf4j-version]
                  [org.slf4j/jul-to-slf4j ~slf4j-version]
                  [org.slf4j/jcl-over-slf4j ~slf4j-version]
-                 [yada "1.1.33" :exclusions [com.fasterxml.jackson.core/jackson-core]]]
+                 [mantree/experimental-yada "1.1.44" :exclusions [com.fasterxml.jackson.core/jackson-core]]
```

Comment:
It's such a small thing, and i think this version of it makes good sense. Hopefully it'll get pulled in at some point